src/Header.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+class Header extends React.Component {
+    constructor(props) {
+        super(props);
+        this.handleWatchList = this.handleWatchList.bind(this);
+    }
+    handleWatchList() {
+        document.querySelectorAll('.movie-info__moreinfo').forEach(item => item.classList.remove("disabled"));
```

Comment:
It would be better to handle this using JSX rather than DOM manipulation as a re-render could blow out your changes and create some strange bugs